src/celeritas/phys/Process.hh

```diff
@@ -63,10 +58,18 @@ class Process
     using VecModel           = std::vector<SPConstModel>;
     using StepLimitBuilders  = ValueGridArray<UPConstGridBuilder>;
     using ActionIdIter       = RangeIter<ActionId>;
-    using ProcessType        = ::celeritas::ProcessType;
     using Applicability      = ::celeritas::Applicability;
     //!@}
 
+    // TODO: update options based on ImportData
+    struct Options
+    {
+        bool use_integral_xs;
+
+        Options(bool integral_xs) : use_integral_xs{integral_xs} {}
+        Options() : Options(false) {}
```

Comment:
Does this not work:
```suggestion
        bool use_integral_xs = false;
```
? 

What other options did you envision here? My first thought is that instead of defining the options struct and type in the base class we should have virtual functions for each datum that needs to be accessed. If we find ourselves duplicating a lot of code we could use a concrete daughter class that implements the common use case.

Reply:
Yeah, I started out with a virtual `use_integral_xs()` function and switched (prematurely) to a more general method in case there were other options we needed. You're right, since we currently only have to access this guy I'll get rid of the base options and just keep a function returning the integral approach flag.